espnet/asr/pytorch_backend/asr.py

```diff
@@ -195,7 +195,9 @@ def __call__(self, batch, device):
         # perform padding and convert to tensor
         xs_pad = pad_list([torch.from_numpy(x).float() for x in xs], 0).to(device)
         ilens = torch.from_numpy(ilens).to(device)
-        ys_pad = pad_list([torch.from_numpy(y).long() for y in ys], self.ignore_id).to(device)
```

Comment:
We can keep the original one because the current speech translation recipe does not use multi-task learning.

Reply:
We always have two references per uttererance, one for source language (ASR) and another for target language (translation). I cannot run training scripts without this modification in the current master branch.

Reply:
I see.